code/modules/factory/parts.dm

```diff
@@ -97,7 +102,7 @@ GLOBAL_LIST_INIT(sadar_ammo_recipe, list(
 /obj/item/factory_part/sadar_ap
 	name = "SADAR AP missile asssembly"
 	desc = "An unfinished sleek missile with an AP warhead."
-	result = /obj/item/ammo_magazine/rocket/sadar/ap
+	result = list(/obj/item/ammo_magazine/rocket/sadar/ap)
```

Comment:
Undo the changes making the single result to lists

Reply:
Done